kyaml/runfn/runfn.go

```diff
@@ -468,11 +468,26 @@ func (r *RunFns) ffp(spec runtimeutil.FunctionSpec, api *yaml.RNode, currentUser
 		if err != nil {
 			return nil, err
 		}
+
+		// Storage mounts can either come from kustomize fn run --mounts,
+		// or from the declarative function mounts field.
+		// In the latter case, we ensure that the mount path is subject
+		// to kustomize's load restrictions.
+		storageMounts := spec.Container.StorageMounts
+		for _, mount := range storageMounts {
+			if strings.HasPrefix(filepath.Clean(mount.Src), "../") {
+				return nil, fmt.Errorf("mount path %s is not permitted; declarative mount paths must be under the current directory", mount.Src) //nolint:goerr113
```

Comment:
What's the linter failure here? Since we're in kyaml, should we be using the errors package instead of fmt anyway?

Reply:
Ah yeah you're right, I'll change it to use `errors`, that should make the linter failure go away. Thanks!